src/Parameter.cpp

```diff
@@ -31,6 +32,7 @@ struct ParameterContents {
         // dense vectorization. You can unset it by setting it to a
         // null expression. (param.set_stride(0, Expr());)
         stride_constraint[0] = 1;
+        host_alignment = 1;
```

Comment:
I could imagine a lazy crop of an 8-bit planar image resulting in 1-byte
alignment of the host pointer.

On Mon, Apr 4, 2016 at 3:36 PM, Steven Johnson <notifications@github.com>
wrote:

> In src/Parameter.cpp
> <https://github.com/halide/Halide/pull/1100#discussion_r58460925>:
>
> > @@ -31,6 +32,7 @@ struct ParameterContents {
> >          // dense vectorization. You can unset it by setting it to a
> >          // null expression. (param.set_stride(0, Expr());)
> >          stride_constraint[0] = 1;
> > +        host_alignment = 1;
>
> out of curiosity: is there any modern system for which we'll see an
> allocator return values that aren't least 2 (or 4) byte aligned?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/1100/files/2d41911c852aa3886d746060c5c4a446bad04459#r58460925>
>